package.json

```diff
@@ -1,17 +1,25 @@
 {
   "name": "eslint-config-canopy",
-  "version": "3.0.0",
+  "version": "4.0.0",
   "description": "A standard eslint configuration for Canopy frontend developers",
-  "main": "lint-config.js",
+  "main": "eslint.config.js",
+  "type": "module",
   "repository": "git@canopy.githost.io:front-end/canopy-eslint.git",
   "license": "Apache-2.0",
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^6.10.0",
-    "@typescript-eslint/parser": "^6.10.0",
-    "@babel/preset-typescript": "^7.23.2",
```

Comment:
is the assumption here that each individual repo will install this on their own?

Reply:
for `@babel/preset-typescript` yes, it's not required for eslint